packages/neo-one-website/tutorial/tutorial.md

```diff
@@ -101,11 +104,11 @@ This smart contract doesn't do a whole lot, in fact it does nothing, but this is
   4. Deploy the smart contracts to the private NEO network.
   5. Generate code for use in your decentralized app.
 
-Additionally, you may run `yarn build` with the `--reset` flag to reset the network. Running with `--watch` will execute the above process whenever you make a change to your smart contracts.
+Additionally, you may run `yarn build` OR `npm build` with the `--reset` flag to reset the network. Running with `--watch` will execute the above process whenever you make a change to your smart contracts.
```

Comment:
This should actually be `yarn neo-one build`

Reply:
👍